.circleci/config.yml

```
@@ -1,20 +1,9 @@
-version: 2
+version: 2.1
 jobs:
   build:
-#    docker:
-#      - image: debian:stretch
     machine: true
-
     steps:
-
       - checkout
-
       - run: docker login -u $DOCKER_USER -p $DOCKER_PASS
-
-      # build the conda version
-#       - run: docker build -t ldssa/conda-base:latest .
-#       - run: docker push ldssa/conda-base:latest
-
-      # build the pip version
-      - run: docker build -t ldssa/pip-base:latest -f Dockerfile-pip .
-      - run: docker push ldssa/pip-base:latest
+      - run: docker build -t ldssa/academy-base:latest .
+      - run: docker push ldssa/academy-base:latest
```

```suggestion
      - run: docker push ldssa/academy-base:latest
      
workflows:
  version: 2
  build_and_push:
    jobs:
      - build:
          filters:
            branches:
              only: main
```

I didn't test this code, does this make sense?
So we don't end up pushing images when they are WIP, let's specify only to push when in `main`.

---

Or better yet, can we run the CI only when we tag a commit?
It seems possible with the `CIRCLE_TAG` environment variable.

This way, we can make proper releases, in this case with tag:`3.10` for example.